index.html

```diff
@@ -1346,6 +1346,18 @@ <h2>
 format using the <code>publicKeyJwk</code> property or as the raw 32-byte
 public key value encoded in Base58 Bitcoin format using the
 <code>publicKeyBase58</code> property. Other Secp256k1 public key formats
+MUST NOT be used.
+            </td>
+          </tr>
+          <tr>
+            <td>
+ed25519
+            </td>
+            <td>
+Ed25519 public key values MUST be encoded in either JSON Web Key (JWK)
+format using the <code>publicKeyJwk</code> property or as the raw 32-byte
+public key value encoded in Base58 Bitcoin format using the
```

Comment:
Each occurrence of `Base58 Bitcoin format` should be linked to (a reference to) the relevant spec/RFC/whatever.

Reply:
https://tools.ietf.org/id/draft-multiformats-multibase-00.html#rfc.appendix.B.3 ?

Reply:
This appears to be a better choice, albeit not a standard in the usual sense, as it actually *says* something -- <https://en.bitcoin.it/wiki/Base58Check_encoding>

Reply:
This was done in the msporny-issue-67 PR.

Reply:
BTW, Base58Check *is not* Base58 BTC -- Base58Check is a bad name and requires a funky double sha256 + checksum algorithm above and beyond base encoding. We are suggesting base58btc. That said, the issue notes that we haven't made a decision here yet, and still need to do so.